vcsutils/webhookparser/testdata/bitbucketcloud/prclosepayload

```diff
@@ -0,0 +1 @@
t--avatars.us-west-2.prod.public.atl-paas.net%2Finitials%2FYI-5.png"}},"type":"user","nickname":"yahavi","account_id":"557058:40514458-78b7-4960-a0bd-2fcd157761fe"},"merge_commit":null,"closed_by":{"display_name":"Yahav Itzhak","uuid":"{1afb3b20-e42f-4cef-9610-765590780396}","links":{"self":{"href":"https://api.bitbucket.org/2.0/users/%7B1afb3b20-e42f-4cef-9610-765590780396%7D"},"html":{"href":"https://bitbucket.org/%7B1afb3b20-e42f-4cef-9610-765590780396%7D/"},"avatar":{"href":"https://secure.gravatar.com/avatar/9680da1674e22a1de17acb19bb233ebf?d=https%3A%2F%2Favatar-management--avatars.us-west-2.prod.public.atl-paas.net%2Finitials%2FYI-5.png"}},"type":"user","nickname":"yahavi","account_id":"557058:40514458-78b7-4960-a0bd-2fcd157761fe"}},"repository":{"scm":"git","website":null,"uuid":"{ba44938d-74fb-41e2-8f0e-fbbee86358e8}","links":{"self":{"href":"https://api.bitbucket.org/2.0/repositories/yahavi/hello-world"},"html":{"href":"https://bitbucket.org/yahavi/hello-world"},"avatar":{"href":"https://bytebucket.org/ravatar/%7Bba44938d-74fb-41e2-8f0e-fbbee86358e8%7D?ts=default"}},"project":{"links":{"self":{"href":"https://api.bitbucket.org/2.0/workspaces/yahavi/projects/HEL"},"html":{"href":"https://bitbucket.org/yahavi/workspace/projects/HEL"},"avatar":{"href":"https://bitbucket.org/account/user/yahavi/projects/HEL/avatar/32?ts=1630824344"}},"type":"project","name":"hello-world","key":"HEL","uuid":"{0e3bc2fd-7733-4b68-881e-11b8f9630efa}"},"full_name":"yahavi/hello-world","owner":{"display_name":"Yahav Itzhak","uuid":"{1afb3b20-e42f-4cef-9610-765590780396}","links":{"self":{"href":"https://api.bitbucket.org/2.0/users/%7B1afb3b20-e42f-4cef-9610-765590780396%7D"},"html":{"href":"https://bitbucket.org/%7B1afb3b20-e42f-4cef-9610-765590780396%7D/"},"avatar":{"href":"https://secure.gravatar.com/avatar/9680da1674e22a1de17acb19bb233ebf?d=https%3A%2F%2Favatar-management--avatars.us-west-2.prod.public.atl-paas.net%2Finitials%2FYI-5.png"}},"type":"user","nickname":"yahavi","account_id":"557058:40514458-78b7-4960-a0bd-2fcd157761fe"},"workspace":{"slug":"yahavi","type":"workspace","name":"Yahav Itzhak","links":{"self":{"href":"https://api.bitbucket.org/2.0/workspaces/yahavi"},"html":{"href":"https://bitbucket.org/yahavi/"},"avatar":{"href":"https://bitbucket.org/workspaces/yahavi/avatar/?ts=1543655805"}},"uuid":"{1afb3b20-e42f-4cef-9610-765590780396}"},"type":"repository","is_private":false,"name":"hello-world"},"actor":{"display_name":"Yahav Itzhak","uuid":"{1afb3b20-e42f-4cef-9610-765590780396}","links":{"self":{"href":"https://api.bitbucket.org/2.0/users/%7B1afb3b20-e42f-4cef-9610-765590780396%7D"},"html":{"href":"https://bitbucket.org/%7B1afb3b20-e42f-4cef-9610-765590780396%7D/"},"avatar":{"href":"https://secure.gravatar.com/avatar/9680da1674e22a1de17acb19bb233ebf?d=https%3A%2F%2Favatar-management--avatars.us-west-2.prod.public.atl-paas.net%2Finitials%2FYI-5.png"}},"type":"user","nickname":"yahavi","account_id":"557058:40514458-78b7-4960-a0bd-2fcd157761fe"}}
```

Comment:
I don't know if it is worth while, but the test files could have `.json` extension making them easier to read if necessary.

Reply:
Good idea. I'll add extensions.

Reply:
I've added extension for all providers except GitHub. For GitHub payload is not a json.